### ISSUE TITLE
enw_aggregate_cumulative throws error when using timestep=month 

### ISSUE THREAD

**Issue by adrian-lison** [epinowcast/epinowcast#426]
**Describe the bug**
Title says it all.

``` 
> enw_aggregate_cumulative(obs, timestep = "month")
Error in unclass(e1) + unclass(e2) : 
  non-numeric argument to binary operator
```

**To Reproduce**
See branch `issue425` for a test I added to `test-enw_aggregate_cumulative.R`.

**Additional context**
I am also a bit unsure about what resolution "month" implies - if that is calendar-based, the resolution would vary from month to month, right? The documentation could be improved here.

**Comment by adrian-lison** [comment]
@seabbs Sorry, I just noticed that `timestep="month"` is not yet supported and that `enw_preprocess_data` throws an error if this is specified. So we should only add such an error to all other user-facing functions as well?

**Comment by adrian-lison** [comment]
And maybe not write in the doc that "month" is an option?

**Comment by seabbs** [comment]
Yeah I think writing an error check and changing the docs is the change that is needed here. Will add for this release.
